cli/local-net

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python3
+
+import subprocess
+import threading
+import sys
+import getopt
+import os
+from random import *
+import time
+import json
+
+colors = {
+    "red": 91,
+    "green": 92,
+    "yellow": 93,
+    "lightPurple": 94,
+    "purple": 95,
+    "cyan": 96,
+    "lightGray": 97,
+}
+worker_colors = {}
+
+
+def config(network, boot_nodes, rpc_nodes):
+    if not os.path.isdir('./local-net'):
+        os.mkdir('./local-net')
+
+    local_net = os.path.realpath('./local-net')
+    taraxa_cli = os.path.dirname(os.path.realpath(__file__)) + '/taraxa'
+
+    boot_nodes_flags = []
+    private_keys = {}
+    public_addresses = {}
+    for i in range(boot_nodes):
+        worker_name = f'boot-node-{i}'
+        public_key = subprocess.check_output(
+            f'{taraxa_cli} account | grep public', shell=True).strip().decode('utf-8').split(' ')[2]
+        private_key = subprocess.check_output(
+            f'{taraxa_cli} account | grep private', shell=True).strip().decode('utf-8').split(' ')[2]
+        public_address = subprocess.check_output(
+            f'{taraxa_cli} account --secret {private_key} | grep address:', shell=True).strip().decode('utf-8').split(' ')[1]
+        private_keys[worker_name] = private_key[2:]
+        public_addresses[worker_name] = public_address
```

Comment:
The public/private keys and addresses have no relation to each other? 
Cause you are running the command multiple times.. I would assume they have some kind of relation.

Reply:
Right. `public_key` should have also been derived from `private_key`. Fixed